src/main/java/com/entrevistador/orquestador/infrastructure/rest/controller/VistaPreviaEntrevistaController.java

```diff
@@ -31,5 +43,18 @@ public List<VistaPreviaEntrevistaDto> crearSolicitudEntrevista() {
 
     }
 
+    @GetMapping(value = "/perfiles")
+    public Flux<SoloPerfilImp> mostrarListaPerfiles(){
+        return entrevistaPrueba.getPerfiles(10);
```

Comment:
Este valor lo podemos quitar de el controlador y dejarlo en el damo como una constante o dejarlo en la misma Dao pero como un propertie por si nos toca cambiarlo solo cambiarlo en el aplication
Mas adelante vamos a aplciar algo llamado configmap yun refres del config map en cloud que no necesitara despliegue del micro y refresca los valores

Reply:
Se utilizó un resourse en el Dao correspondiente para el valor constante
`
    private final EntrevistaRepository entrevistaRepository;
    private final EntrevistaPruebaConst entrevistaPruebaConst;

    @Override
    public Flux<SoloPerfilImp> getPerfil() {
        return this.entrevistaRepository.obtenerPerfilEmpresa(entrevistaPruebaConst.getLimitperfiles());
    }
`